internal/pkg/cli/svc_init.go

```diff
@@ -220,13 +222,22 @@ func (o *initSvcOpts) Execute() error {
 		}
 	}
 
+	o.os, o.arch, err = o.getOSArch()
```

Comment:
hmm what if they don't have a docker daemon running? Will it break since I guess they can use Copilot without docker running with existing image?

Reply:
Yes. 
It errors out with : 
```
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
✘ get os/arch from docker: get docker os: exit status 1
```
But you're right-- they should be allowed to do this. Thanks for catching this. Will fix!